server/sandbox_remove.go

```diff
@@ -74,27 +71,10 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		}
 	}
 
-	if err := label.ReleaseLabel(sb.processLabel); err != nil {
-		return nil, err
-	}
-
-	// unmount the shm for the pod
-	if sb.shmPath != "/dev/shm" {
-		if mounted, err := mount.Mounted(sb.shmPath); err == nil && mounted {
-			if err := syscall.Unmount(sb.shmPath, syscall.MNT_DETACH); err != nil {
-				return nil, err
-			}
-		}
-	}
-
-	if err := sb.netNsRemove(); err != nil {
-		return nil, fmt.Errorf("failed to remove networking namespace for sandbox %s: %v", sb.id, err)
-	}
-
 	s.removeContainer(podInfraContainer)
```

Comment:
Just wondering if a comment hear stating that a stop must be done before the remove is called.  Just as a bread crumb in case someone later is trying to figure things out when memory is leaking and the ordering isn't being held up as it should.  Your call, just a thought.